project-api-tests/src/test/java/org/ballerina/projectapi/CentralTestUtils.java

```diff
@@ -199,4 +229,148 @@ static Path getBalaPath(Path projectPath, String org, String pkgName, String pla
+                                           Map<String, String> envVariables) throws IOException, InterruptedException {
+        Process search = executeSearchCommand(DISTRIBUTION_FILE_NAME,
+                tempWorkspaceDirectory,
+                new LinkedList<>(Collections.singletonList(pkg)),
+                envVariables);
+        String buildErrors = getString(search.getErrorStream());
+        if (!buildErrors.isEmpty()) {
+            Assert.fail(OUTPUT_CONTAIN_ERRORS + buildErrors);
+        }
+        String buildOutput = getString(search.getInputStream());
+        return buildOutput;
+    }
+
+    /**
+     * Build bala for a package.
+     *
+     * @param tempWorkspaceDirectory Path to workspace
+     * @param envVariables Environmental variables
+     * @param packageName Package
+     * @param orgName Organization name
+     * @param version Package Version
+     * @param additionalArgs Additional arguments to pass wben building package.
+     * @throws IOException
+     * @throws InterruptedException
+     */
+    static void buildPackageBala(Path tempWorkspaceDirectory, Map<String, String> envVariables, String packageName,
+                             String orgName, String version, List<String> additionalArgs) throws IOException, InterruptedException {
+        List<String> argsCollection = new ArrayList(additionalArgs);
+        argsCollection.add("-c");
+        Process build = executeBuildCommand(DISTRIBUTION_FILE_NAME,
+                tempWorkspaceDirectory.resolve(packageName),
+                new LinkedList<>(argsCollection), envVariables);
+        String buildErrors = getString(build.getErrorStream());
+        if (!buildErrors.isEmpty()) {
+            Assert.fail(OUTPUT_CONTAIN_ERRORS + buildErrors);
+        }
+        String buildOutput = getString(build.getInputStream());
+        if (!buildOutput.contains(getGenerateBalaLog(orgName,
+                packageName, ANY_PLATFORM, version))) {
+            Assert.fail(OUTPUT_NOT_CONTAINS_EXP_MSG + getGenerateBalaLog(orgName,
+                    packageName, ANY_PLATFORM, version));
+        }
+        Assert.assertTrue(
+                getBalaPath(tempWorkspaceDirectory.resolve(packageName), orgName, packageName, ANY_PLATFORM,
+                        version).toFile().exists());
+    }
+
+    /**
+     * Push a bala to remote repository.
+     *
+     * @param tempWorkspaceDirectory Path to workspace
+     * @param projectName Project name
+     * @param envVariables Environmental variables
+     * @param orgName Organization name
+     * @param packageName package
+     * @param version Package Version
+     * @throws IOException
+     * @throws InterruptedException
+     */
+    static void testPushPackage(Path tempWorkspaceDirectory, String projectName, Map<String, String> envVariables,
+                                String orgName, String packageName, String version) throws IOException, InterruptedException {
```

Comment:
Line is too long

Reply:
There are few places to correct

Reply:
Why are these styling issues not getting validated? It's better to add early as possible right?

Reply:
Looks like we haven't included check style plugin in the project. Will be incorporating it. That's the reason for missing these styling issues.